StereoKitC/systems/render.cpp

```diff
@@ -878,9 +893,34 @@ void render_blit(tex_t to, material_t material) {
 
 ///////////////////////////////////////////
 
+void render_save_to_file(color32* color_buffer, int width, int height, void* context) {
+	stbi_write_jpg((char*)context, width, height, 4, color_buffer, 90);
+	sk_free(context);
+}
+
 void render_screenshot(const char *file, vec3 from_viewpt, vec3 at, int width, int height, float fov_degrees) {
 	char *file_copy = string_copy(file);
-	render_screenshot_list.add( render_screenshot_t{ file_copy, from_viewpt, at, width, height, fov_degrees });
+	matrix view = matrix_trs(
+		from_viewpt,
+		quat_lookat(from_viewpt, at));
+	matrix_inverse(view, view);
+	matrix proj = matrix_perspective(fov_degrees, (float)width / height, render_clip_planes.x, render_clip_planes.y);
+	render_screenshot_list.add( render_screenshot_t{ render_save_to_file, file_copy, view, proj, rect_t(), width, height, render_layer_all, render_clear_all });
```

Comment:
Definitely not nit picky! I just read about how its ambiguity and how it may even lead to the "most vexing parse" :o plus it's recommended all t/o the [docs](https://learn.microsoft.com/en-us/cpp/cpp/initializing-classes-and-structs-without-constructors-cpp?view=msvc-170). Thanks for bringing this up!